scripts/griddle.jsx

```diff
@@ -153,6 +154,54 @@ var Griddle = React.createClass({
            return false;
        });
     },
+
+    filterByColumnFilters(columnFilters) {
+      var filteredResults = Object.keys(columnFilters).reduce(function(previous, current) {
+        return _filter(
+          previous,
+          function(item) {
+            if(deep.getAt(item, current || "").toString().toLowerCase().indexOf(columnFilters[current].toLowerCase()) >= 0) {
+              return true;
+            }
+
+            return false;
+          }
+        )
+      }, this.props.results)
+
+      var newState = {
+        columnFilters: columnFilters
+      };
+
+      if(columnFilters) {
+        newState.filteredResults = filteredResults;
+        newState.maxPage = this.getMaxPage(newState.filteredResults);
+      } else if(this.state.filter) {
+        newState.filteredResults = this.props.useCustomFilterer ?
+          this.props.customFilterer(this.props.results, filter) :
+          this.defaultFilter(this.props.results, filter);
+      } else {
+        newState.filteredResults = null;
+      }
+
+      this.setState(newState);
+    },
+
+    filterByColumn: function(filter, column) {
```

Comment:
Here we're defining the column filters. The `filterByColumnFilters` method takes all the columnFilters and sorts the data by them -- this only sets them and calls the other function.